core/node/main/main.cpp

```diff
@@ -212,7 +213,7 @@ namespace fc {
         [&]() -> outcome::result<std::vector<Import>> {
       return node_objects.storage_market_import_manager->list();
     };
-
+    OUTCOME_TRY(token, node_objects.api->AuthNew(kAllPermission));
```

Comment:
either `.value()` to throw, or log right here, otherwise detecting error source will be difficult